itstool.in

```diff
@@ -1577,7 +1577,7 @@ if __name__ == '__main__':
             if opts.output == '-':
                 out = sys.stdout
             else:
-                out = open(opts.output, 'w')
+                out = open(opts.output, 'wb')
```

Comment:
This is the one that makes it work; the other two I changed as it was the only places left opening files not 'wb'

Reply:
OK, but sys.stdout is still not-binary ;) 
I don't get idea why we need to use bytes instead of strs. I propose to comment encode() on py3.6